benchmark/wildtime_benchmarks/example_pipelines/data_drift_trigger/huffpost_datadrift.yaml

```diff
@@ -46,6 +46,7 @@ trigger:
   metrics:
     ev_mmd:
       id: AlibiDetectMmdDriftMetric
+      num_permutations: 1000
```

Comment:
do we want to do permutation testing after all?

Reply:
or is this just to make the unit tests happy?

Reply:
this was just to make the unit tests happy, since this an example pipeline. these are not the experiments i want to run :D